Mathlib/Algebra/Quaternion.lean

```diff
@@ -64,45 +64,51 @@ structure QuaternionAlgebra (R : Type*) (a b : R) where
   imK : R
 
 @[inherit_doc]
-scoped[Quaternion] notation "ℍ[" R "," a "," b "]" => QuaternionAlgebra R a b
-open Quaternion
+scoped[Quaternion] notation "ℍ[" R "," a "," b "," c "]" =>
+    QuaternionAlgebra R a b c
+
+@[inherit_doc, deprecated "This is a deprecated expression for QuoternionAlgebra R a 0 b,
+  it's only true when charateristic not equal to 2, please use the general version
+  ℍ[R,a,0,b] instead. " (since := "2025-01-15")]
```

Comment:
I don't think we need this level of detail here; perhaps
```suggestion
@[inherit_doc, deprecated "Use ℍ[R,a,0,b] instead of ℍ[R,a,b]." (since := "2025-01-15")]
```